Source/include/Ably/ARTRealtime.h

```diff
@@ -25,9 +25,9 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- The protocol upon which the top level object `ARTRealtime` is implemented.
+ This protocol contains the non-initializer instance methods provided by the `ARTRealtime` client class.
  */
-@protocol ARTRealtimeProtocol <NSObject>
+@protocol ARTRealtimeInstanceMethodsProtocol <NSObject>
```

Comment:
Could also be `ARTRealtimeInstanceProtocol` or `ARTRealtimePrincipalProtocol`, just a minor suggestion.

Reply:
Thanks for the suggestions! I don't think that any one of the three is loads better than the others, so will keep as is 🙂